src/libYARP_math/include/yarp/math/FrameTransform.h

```diff
@@ -13,121 +13,70 @@
 #include <yarp/math/Math.h>
 #include <yarp/math/Quaternion.h>
 
+#define VALID_POSITION         1  // 0b00000001
+#define VALID_ROTATION         2  // 0b00000010
+#define VALID_LIN_VELOCITY     4  // 0b00000100
+#define VALID_LIN_ACCELERATION 8  // 0b00001000
+#define VALID_ANG_VELOCITY     16 // 0b00010000
+#define VALID_ANG_ACCELERATION 32 // 0b00100000
+#define VALID_FULL VALID_POSITION | VALID_ROTATION | VALID_LIN_VELOCITY | VALID_LIN_ACCELERATION | VALID_ANG_VELOCITY | VALID_ANG_ACCELERATION
+
```

Comment:
I agree with @barbalberto on moving them to an enum. I would avoid preprocessor macros for these use cases.